slack/slack.go

```diff
@@ -207,7 +207,13 @@ func (s *slackInterface) SendChannelNotification(ctx context.Context, chanID, me
 }
 
 func (s *slackInterface) SendChannelReport(ctx context.Context, chatID, title string, items []string) (messages.MessageHandle, error) {
-	return s.PostMessage(ctx, chatID, fmt.Sprintf("*%s*\n\n%s", title, strings.Join(items, "\n\n")))
+	titleHandle, err := s.PostMessage(ctx, chatID, fmt.Sprintf("*%s*", title))
+	if err != nil {
+		return titleHandle, errs.Wrap(err)
+	}
+	mh := titleHandle.(*messageHandle)
+	_, err = s.PostMessageThread(ctx, chatID, mh.Timestamp, strings.Join(items, "\n\n"))
```

Comment:
Oh ok. Good idea, moving it to a thread. If we do that, do you think it might also be worthwhile to separate each line into its own message on the thread?

Reply:
I don't have a strong opinion. It don't think it would help that much and slack would end up then putting sender & time for each message.

Reply:
The (small) benefit would be that your notifications from it would show the line(s) that actually apply to you, instead of the prefix of the whole wall of text

Reply:
In that case, sure, we can try that. If it causes problems we can of course later go to a single message.